sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -2347,6 +2349,31 @@ public partial class AdditionalUnattendContent
         public static bool operator !=(Azure.ResourceManager.Compute.Models.AggregatedReplicationState left, Azure.ResourceManager.Compute.Models.AggregatedReplicationState right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class AlternativeOption
+    {
+        public AlternativeOption() { }
+        public Azure.ResourceManager.Compute.Models.AlternativeType? AlternativeType { get { throw null; } set { } }
+        public string Value { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct AlternativeType : System.IEquatable<Azure.ResourceManager.Compute.Models.AlternativeType>
```

Comment:
the name of this model is too generic. Could we change this to a more descriptive name?